apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.key === 'Control') {
```

Comment:
@egorlevichev  this code is going to be modified when softkey will be integrated.